Xamarin.PropertyEditing/ViewModels/PropertyViewModel.cs

```diff
@@ -77,6 +78,32 @@ public override Resource Resource
 			}
 		}
 
+		public bool SupportsAutocomplete
+		{
+			get { return this.supportsAutocomplete; }
+			private set
+			{
+				if (this.supportsAutocomplete == value)
+					return;
+
+				this.supportsAutocomplete = value;
+				OnPropertyChanged();
+
+				if (!value) {
+					this.autocomplete = null;
+					this.autocompleteCancel?.Cancel();
+					this.autocompleteCancel = null;
+				}
+			}
+		}
+
+		public IReadOnlyList<string> AutocompleteItems => this.autocomplete;
+
+		public string PreviewCustomExpression
```

Comment:
Write-only property is weird. Why not just call `UpdateAutoComplete` directly instead?

Reply:
Binding-ease.